SConsProject/scons
# Update of the documentation toolchain to work properly under Python3

This PR reverts some changes by @mwichmann (sorry!) and removes the libxslt support from the Docbook Tool completely. It should fix all problems for (re-)creating the docs as needed for a full SCons release (see also the discussions in #3580 and related).

I have verified that all basic steps

- validation,
- update of the generated MOD/GEN files,
- update of the scons_example outputs, as well as
- creating the documentation itself with `SKIP_DOC=1 python3 scripts/scons.py build`

work without any problems in the 'build' Docker images (my reference in this case) for Ubuntu 19.10 and Fedora 32 (see folder `testing/docker`).

## Contributor Checklist:

* [ ] I have created a new test or updated the unit tests to cover the new/changed functionality.
* [x] I have updated `CHANGES.txt` (and read the `README.rst`)
* [x] I have updated the appropriate documentation
